scripts/clang-tidy-wrapper.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
+
+set -euo pipefail
+
+CLANG_TIDY="${1}"
+CMAKE_BINARY_DIR="${2}"
+CMAKE_SOURCE_DIR="${3}"
+FILE="${4}"
+
+if [[ ! -e "${CMAKE_BINARY_DIR}/compile_commands.json" ]]; then
+  echo "compile_commands.json not found!"
+  exit 1
+fi
+
+# Filter out anything that we aren't trying to compile. Older clang-tidy
+# versions (<=6) skipped them automatically.
+if grep "${FILE}" "${CMAKE_BINARY_DIR}/compile_commands.json"; then
```

Comment:
```suggestion
if grep -q "${FILE}" "${CMAKE_BINARY_DIR}/compile_commands.json"; then
```

Reply:
Oops 2 seconds too late :)

Reply:
Sorry! Now I see what's going on. Wanna make a PR for that or shall I?

Reply:
Ok I'll do it quick